src/fragments/lib/logging/common/change_log_levels/change_log_levels.mdx

```diff
@@ -0,0 +1,121 @@
+## Change Default Log Level
+
+Below is an example of setting the default log level to `WARN`:
+
+import iosDefaultLogLevel from '/src/fragments/lib/logging/ios/change_log_levels/10_default_log_level.mdx';
+import androidDefaultLogLevel from '/src/fragments/lib/logging/android/change_log_levels/10_default_log_level.mdx';
+import commonDefaultLogLevel from '/src/fragments/lib/logging/common/change_log_levels/10_default_log_level.mdx';
+
+<BlockSwitcher>
+    <Block name="With Configuration File">
+        <Fragments fragments={{all: commonDefaultLogLevel}}/>
+    </Block>
+    <Block name="Without Configuration File">
+         <Fragments
+            fragments={{
+                ios: iosDefaultLogLevel,
+                android: androidDefaultLogLevel
+            }}/>
+    </Block>
+</BlockSwitcher>
+
+## Change local storage maximum size
+To change the maximum amount of local logs are stored before sending logs to CloudWatch.
+
+Below is an example of setting the local storage size to 2 MB:
+
+import iosLocalStorageSize from '/src/fragments/lib/logging/ios/change_log_levels/20_local_storage_size.mdx';
+import androidLocalStorageSize from '/src/fragments/lib/logging/android/change_log_levels/20_local_storage_size.mdx';
+import commonLocalStorageSize from '/src/fragments/lib/logging/common/change_log_levels/20_local_storage_size.mdx';
+
+<BlockSwitcher>
+    <Block name="With Configuration File">
+        <Fragments fragments={{all: commonLocalStorageSize}}/>
+    </Block>
+    <Block name="Without Configuration File">
+        <Fragments
+            fragments={{
+                ios: iosLocalStorageSize,
+                android: androidLocalStorageSize
+            }}/>
+    </Block>
+</BlockSwitcher>
+
+## Change automatic log flush interval
+To change the time interval when logs are automatically flushed and sent to CloudWatch.
```

Comment:
can we please move this section to the "Flush logs" page.

Reply:
Section moved to the flush log page.